### PR TITLE
Adding the icons for the toggle option

### PR DESCRIPTION
**Pull request by Erikang20**
Creating the icons section to toggle the light and dark theme.
Screenshot 2024-07-03 at 1 26 07 PM

### CODE REVIEW

File: client/src/components/Sidebar/index.jsx
Comment (Monika5S) [inline client/src/components/Sidebar/index.jsx:1]: Hi, I noticed that the icon widths are different in the light and dark mode

Comment (Erikang20) [reply client/src/components/Sidebar/index.jsx:1]: You are right. However, I'll be addressing light vs dark looks in upcoming PRs. I wanted to ship the component for now :)

Comment (Monika5S) [reply client/src/components/Sidebar/index.jsx:1]: Sounds good, thanks!

**Review by Monika5S — APPROVED** [review]
Looks good to me. Ready to approve.

**Pull request merged**